src/callstack.cr

```diff
@@ -145,14 +145,20 @@ struct CallStack
   end
 
   private def decode_backtrace
+    current_dir = Dir.current
+    current_dir += '/' unless current_dir.ends_with?('/')
+
     @callstack.compact_map do |ip|
       pc = CallStack.decode_address(ip)
 
       file, line, column = CallStack.decode_line_number(pc)
-      if file == "??"
-        file_line_column = "??"
-      else
+
+      if file && file != "??"
         next if @@skip.includes?(file)
+
+        # Turn to relative to the current dir, if possible
+        file = file.lchop(current_dir)
+
         file_line_column = "#{file} #{line}:#{column}"
```

Comment:
I'd like to have <code>"#{file}<b><u>:</u></b>#{line}:#{column}"</code> because my text editor understands when you tell it to open `foo.cr:7:4`.

Reply:
Oh, I thought about changing that to be like that but thought it would be harder on the eyes, and easier to copy just the filename. But yes, if it's easier to open than on an editor, let's change it (Ruby also uses that format)